### PR TITLE
Dell ztp dev

### PR DESCRIPTION
**Pull request by arunsundark**
ztpcli

Note: The files may some openconfig platform stuff. They are place holders until the sonic-yangs with rpc keyword is recognized(ref pic below) and compiled properly.

rpcyang error

### CODE REVIEW

**Comment by amybuck** [comment]
Has the command changed from config ztp [enable|disable]?

File: src/CLI/actioner/sonic-cli-ztp.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                     print "%Error: Transaction Failure"
+                     return
+            print "%Error: Transaction Failure"
+        else:
+            print "Failed"
+
+if __name__ == '__main__':
+
+    pipestr().write(sys.argv)
+    #pdb.set_trace()
```
Comment (justinejose91) [inline src/CLI/actioner/sonic-cli-ztp.py:110]: Please remove this line.

**Review by justinejose91 — APPROVED** [review]

**Comment by arunsundark** [comment]
> Has the command changed from config ztp [enable|disable]?

The command remains config ztp [enable|disable] in the host, but it  is ztp [enable|disable] inside the docker.

File: src/CLI/actioner/sonic-cli-ztp.py (hunk 1 above)
Comment (arunsundark) [reply src/CLI/actioner/sonic-cli-ztp.py:110]: Sure

**Pull request merged**